person.rb

```diff
@@ -1,22 +1,35 @@
 require_relative 'nameable'
+require_relative 'classroom'
+require_relative 'book'
+require_relative 'rental'
 
 class Person < Nameable
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rental
 
-  attr_reader :id
+  attr_reader :id, :classroom
 
   def initialize(age, name = 'unknown', parent_permission: true)
     super()
     @id = Random.rand(1..100)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rental = []
   end
 
   def correct_name
     @name
   end
 
+  def classroom=(classroom)
+    @classroom = classroom
+    classroom.add_student(self)
+  end
+
```

Comment:
- [ ] You should make sure that the `belongs-to` relationship between Classroom and Student is created in `student.rb` file, so when setting the classroom for a student it also adds it to the classrooms' students. You should have a method that serves as the 'belongs-to' side (a student belongs to a classroom)` 🙃 

It can be similar to 👇🏾 

```ruby
def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```